src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -401,14 +401,18 @@ private string GetPackageDeprecationAlternatePackageText(AlternatePackageMetadat
             {
                 text = string.Format(
                     CultureInfo.CurrentCulture,
-                    Resources.Label_DeprecatedPackageIdAndMinVersion,
+                    Resources.Label_DeprecatedPackageIdAndExactVersion,
                     alternatePackageMetadata.PackageId,
                     alternatePackageMetadata.Range.MinVersion.ToNormalizedString());
             }
 
             if (!alternatePackageMetadata.Range.HasUpperBound)
             {
-                text += Resources.Label_OrHigher;
+                text = string.Format(
+                    CultureInfo.CurrentCulture,
+                    Resources.Label_DeprecatedPackageIdAndMinVersion,
+                    alternatePackageMetadata.PackageId,
+                    alternatePackageMetadata.Range.MinVersion.ToNormalizedString());
```

Comment:
How about VersionRange.ToLockFileDependencyGroupString? 

That does the whole >= on it's own. 

It also would handle all the ranges correctly if needed.

Reply:
That API doesn't exist on `VersionRange` type. It exists on `LibraryRange`.

Reply:
Much better though: https://github.com/NuGet/NuGet.Client/pull/2939/commits/3568133c71859c0cae5e8e6cdc3a5c12b0ee9a1e

Introduced `VersionRange.ToPackageVersionRangeSpecificationString(string packageId)` and forwarded the calls.

Reply:
How does PrettyPrint compare to your change?

I'd imagine it's almost identical :) 
Save for the `()`

Reply:
On another note, should that string be added to the VersionRangeFormatter like the rest of the string representations of the Version Range.

Reply:
Hmm... `PrettyPrint` might actually do the trick IMO. See below.

![image](https://user-images.githubusercontent.com/880728/61617059-3f4c8100-ac6a-11e9-84ba-bc2d40f2f05d.png)

Reply:
Reverted to use `PrettyPrint` instead :) https://github.com/NuGet/NuGet.Client/pull/2939/commits/03838cdb06c60782d11e46ed60e7266125924609